extensions/registration-service/src/test/java/org/eclipse/dataspaceconnector/registration/store/InMemoryParticipantStoreTest.java

```diff
@@ -61,4 +61,14 @@ void listParticipantsWithStatus() {
         store.save(participant03);
         assertThat(store.listParticipantsWithStatus(AUTHORIZED)).containsOnly(participant01, participant03);
     }
+
+    @Test
+    void saveAndFindByDid() {
+        assertThat(store.findByDid(participant1.getDid())).isNull();
+
+        store.save(participant1);
+
+        var participantOptional = store.findByDid(participant1.getDid());
```

Comment:
confusing naming, as this is not an `Optional` instance

Reply:
Nice catch, was there do an older change.

Fixed now. Thanks